docs/browser-bindings/browser/README.md

```diff
@@ -1,6 +1,13 @@
+# @0x/mesh-browser - v1.0.0
```

Comment:
Is there a specific reason to add `v1.0.0` here? AFAIK we only need to do that in __package.json__ because Yarn requires it. I would prefer not to include version numbers anywhere in the `development` branch. If we must include them, it should either be "development" or the commit hash.

Reply:
Since the version are `1.0.0`, `@0x/ts-docs-gen` (which uses `typedoc` under-the-hood) will use that in the title. We could add a script to doc generation if you think it makes sense.

As far as using `1.0.0` in the `package.json`, that is what I needed to do to use `yarn install`, but I will double-check this.